models/articles.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection');
+
+exports.fetchArticleById = (article_id) => {
+    return db
+        .query(
+            `SELECT author, title, article_id, body, topic, created_at, votes, article_img_url
+             FROM articles
+             WHERE article_id = $1`,
+            [article_id]
+        )
+        .then((result) => {
+            if (result.rows.length === 0) {
```

Comment:
Think about when you read an article on bbc news - if the article has no comments, you dont want to see an error just because nobody has commented on it right? An error will prevent us from loading the page and reading the article - in which case no new article would ever receive a comment because we'd never be able to read it in the first place! So you want to say the request for the comments was successful - you just didnt get any comments back - so we'll need a 200 test for no comments returned!

In which case your 404 test for this endpoint will fail - how could we fix that?